packages/sdk-node/src/clerkExpressRequireAuth.ts

```diff
@@ -43,7 +43,25 @@ export const createClerkExpressRequireAuth = (createOpts: CreateClerkExpressMidd
         return handleUnknownCase(res, requestState);
       }
       if (requestState.isInterstitial) {
-        const interstitial = await clerkClient.remotePrivateInterstitial();
+        let interstitial;
+
+        /**
+         * This is a step for deprecating the usage of `remotePrivateInterstitial`
```

Comment:
🙃 If the PK is unavailable in Express, we can't get a local interstitial. So, for the future reader, the comment should state that this is an improvement if we have PK or FrontendAPI key to avoid an extra network call. Mentioning deprecation is a bit confusing.